src/main/scala/wdl4s/values/WdlArray.scala

```diff
@@ -37,7 +37,12 @@ case class WdlArray(wdlType: WdlArrayType, value: Seq[WdlValue]) extends WdlValu
     wdlType.memberType match {
       case t: WdlPrimitiveType => Success(value.map(_.valueString).mkString("\n"))
       case WdlObjectType => WdlObject.tsvSerializeArray(value map { _.asInstanceOf[WdlObject] })
-      case _ => Failure(new UnsupportedOperationException("Can only TSV serialize an Array[Primitive] or Array[Object]"))
+      case WdlArrayType(t: WdlPrimitiveType) =>
```

Comment:
I'm just going to say that you know what I'm going to say here but feel free to ignore me